vendor/bat-native-ads/src/bat/ads/internal/ad_events/ad_event_unittest_util.cc

```diff
@@ -17,41 +20,30 @@
 namespace ads {
 
 AdEventInfo BuildAdEvent(const CreativeAdInfo& creative_ad,
-                         const ConfirmationType confirmation_type,
+                         const AdType& ad_type,
+                         const ConfirmationType& confirmation_type,
                          const base::Time& created_at) {
   AdEventInfo ad_event;
-  ad_event.uuid = base::GenerateGUID();
+  ad_event.type = ad_type;
   ad_event.confirmation_type = confirmation_type;
+  ad_event.uuid = base::GenerateGUID();
+  ad_event.campaign_id = creative_ad.creative_instance_id;
```

Comment:
Should we initialize this with campaign_id and creative_set_id ?

Reply:
Fixed. Great spot :-)